platform/windows/gl_manager_windows.cpp

```diff
@@ -116,13 +125,24 @@ Error GLManager_Windows::_create_context(GLWindow &win, GLDisplay &gl_display) {
 		return ERR_CANT_CREATE; // Return FALSE
 	}
 
-	gl_display.hRC = wglCreateContext(hDC);
+	return OK;
+}
+
+Error GLManager_Windows::_create_context(GLWindow &win, GLDisplay &gl_display) {
+	Error result = _configure_pixel_format(win.hDC);
+	if (result != OK) {
+		return result;
+	}
```

Comment:
Nitpick, but we usually use `err` for error codes.

Reply:
I agree, that is a nitpick.

Reply:
changed